Bottom: 1f293108ce283eca535112af13051a53881e8db9
Top:    63d9ee86b7812695228cc4f4018bf998f3eaaa39
Author: Sean Hefty <sean.hefty@intel.com>
Date:   2013-02-04 16:50:19 -0800

Refresh of kern-bug

---

diff --git a/src/cma.c b/src/cma.c
index 918592a..2fb9913 100755
--- a/src/cma.c
+++ b/src/cma.c
@@ -48,6 +48,7 @@
 #include <byteswap.h>
 #include <stddef.h>
 #include <netdb.h>
+#include <syslog.h>
 
 #include "cma.h"
 #include "indexer.h"
